offer-manager-domain/src/main/groovy/life/qbic/business/offers/TaxOffice.groovy

```diff
@@ -0,0 +1,67 @@
+package life.qbic.business.offers
+
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.datamodel.dtos.business.AffiliationCategory
+import life.qbic.datamodel.dtos.business.services.ExternalServiceProduct
+
+
+/**
+ * <b>Tax office that is responsible for offer tax calculation.</b>
+ *
+ * <p>Calculates the taxes that need to be considered for a given price value, affiliation and product type</p>
+ *
+ * <p>The current implementation takes a VAT ratio of 19%, default for Germany. Taxes only apply for customers within Germany
+ * and if they are not part of the University or UKT. Taxes are not forwarded internally.</p>
+ *
+ * <p>The only exception are products (services) external services, for which taxes need to
+ * be applied regardless of the affiliation status.</p>
+ *
+ * @since 1.2.0
+ */
+class TaxOffice {
+
+    private static final BigDecimal VAT_RATIO = BigDecimal.valueOf(0.19)
```

Comment:
I hear @KochTobi saying: you should always create the BigDecimal out of a string to obtain the correct value

```suggestion
    private static final BigDecimal VAT_RATIO = BigDecimal.valueOf("0.19")
```

Reply:
nvm I just realised you used the valueOf method. So I guess then its fine 🤔